doc/src/guides/solving.md

```diff
@@ -0,0 +1,45 @@
+# Solve Using the SymPy Python Library
+
+The Python package SymPy can symbolically solve equations, differential equations, 
+linear equations, nonlinear equations, matrix problems, inequalities, 
+Diophantine equations, and evaluate integrals. SymPy can also solve numerically.
+
+Learn how to use SymPy computer algebra system to:
+
+| Description                                                  | Example                                                                                                                     |
+|--------------------------------------------------------------|-----------------------------------------------------------------------------------------------------------------------------|
+| [ Solve an equation algebraically ](http://docs.sympy.org)                        | [  $x^2 = 4 \to x =[-2, 2] $  ](http://docs.sympy.org)                                                                                                       |
+| [ Solve a system of equations algebraically ](http://docs.sympy.org)              | [  $x^2 + y = 2, x - y = 4 \to [(x = -3, y = -7), (x = 2, y = 2)]$  ](http://docs.sympy.org)                                                                                              |
+| [ Solve an equation numerically ](http://docs.sympy.org)                          | [ $cos(x) = x \to x \approx 0.739085133215161$ ](http://docs.sympy.org)                                                                                                     |
+| [ Solve a partial differential equation algebraically ](http://docs.sympy.org)    | [${\frac {\partial ^{2}(x^3 y^3)}{\partial x\partial y}}=9 \to x \ne 0, y = \pm\frac{1}{x} $](http://docs.sympy.org)                                                                     |
+| [ Solve an ordinary differential equation algebraically  ](http://docs.sympy.org) | [ $f\prime\prime(x) + 9f(x)=0 \to f(x)=C_{1} sin(3x)+ C_{2} cos(3x)$ ](http://docs.sympy.org)                                                    |
```

Comment:
This should just be `'` instead of `\prime`. See https://tex.stackexchange.com/questions/87134/what-is-the-advantage-of-using-f-prime-instead-of-f. If you do use `\prime`, it should be `^\prime`. Right now the prime symbols are too low and look odd.

Reply:
This should be `\sin` and `\cos`

Reply:
Also let's use `y'` instead of `f'(x)` so that it matches what people are more likely to encounter in a textbook.